Gemfile

```diff
@@ -29,5 +29,6 @@ gem 'metadata-json-lint'
 gem 'kafo_module_lint'
 gem 'beaker-rspec', {"groups"=>["system_tests"]}
 gem 'beaker-puppet_install_helper', {"groups"=>["system_tests"]}
+gem 'facterdb', :git => 'https://github.com/camptocamp/facterdb', :ref => 'master'
```

Comment:
This file is modulesynced, so ideally should be released first to avoid this modification.